mail_activity_board/static/src/xml/inherit_chatter.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-extend="mail.Chatter.Buttons">
+         <t t-jquery="button.o_chatter_button_schedule_activity" t-operation="after">
+            <button t-if="schedule_activity_btn" class="btn btn-sm btn-link o_chatter_button_list_activity">
+                 <span class="o_list_activity label label-primary"/> <t t-if="isMobile">List</t><t t-else="">Activities list</t>
```

Comment:
As I have already mentioned to you, when I have to do something, I try to look at how it is done in another part of the code and imitate it.
 
I thought that instead of using an icon, I could use the number of activities for it, in the style of followers and the number of activities when the list of activities is minimized.

That's why I've put the following in the code, besides allowing me to pass the amount in view:

<span class = "o_list_activity label label-primary" />

If you think I should remove it, tell me, but for the time being, until the merge that you mention to me is not done, there is no other way to show the amount.

This causes that at the moment I have to overwrite the render method. To get the amount to that span tag.